commit ce1d97a19c85b1c9a1d74b0cf900f571a8285eea
Author: andymule <amun@tobii.com>
Date:   Thu Aug 22 17:19:38 2019 +0200

    asdasdwq
    X

diff --git a/NewFile b/NewFile
index 579b8dd..3b4a016 100644
--- a/NewFile
+++ b/NewFile
@@ -1,10 +1,10 @@
 This is a new fieThis is a new fileThis is a new fileThis is a new file
-
-This is a new file
+asd
 This is a new file
+Thi is a new file
 I dont like this
 
 
 This is a new file
-This is a file
+This ixs a file
 This is a new file
diff --git a/test2 b/test2
index 31e2258..3496079 100644
--- a/test2
+++ b/test2
@@ -1,26 +1,21 @@
 asldkjasdlkjasd
 asda
 sdas
-da
+daaaa
 
 
 
 
 s
 
-d
-
-d
+dx
 
 d
-
-
-
 a
 
 s
 
 d
 
 
 
